index.html

```diff
@@ -0,0 +1,266 @@
+                            </li>
+                            <li>
+                                <a class="bottom-header-ul-li-text transition font" href="#">Join</a>
+                            </li>
+                            <li>
+                                <a class="bottom-header-ul-li-text transition font" href="#">Sponsor</a>
+                            </li>
+                            <li>
+                                <a class="bottom-header-ul-li-text transition font" href="#">News</a>
+                            </li>
+                           </ul>
+                           <button type="button" class="campaign transition font">
+                            ComdeyBurst Initiative
+                           </button>
+                        </section>
+                    </div>
+                </section>
+            </nav>
+        </div>
+        <div class="page-header-body">
+            <h1 class="header-text-head font for-phone " >"Hey! Comic Lovers"</h1>
+            <h1 class="header-text-head-desk font for-desktop " >"Hey! Comic Lovers"</h1>
+            <h2 class="header-text-body font for-phone">
+                FIND THE BEST
+                <br>
+                CONCERT WHEREVER
+                <br>
+                YOU ARE
+            </h2>
+            <h2 class="header-text-body-desk font for-desktop">
+                FIND YOUR PERFECT
+                <br>
+                COMIC WHEREVER
+                <br>
+                YOU ARE
+            </h2>
+            <section class="header-text-description-container for-phone">
+                <p class="header-text-description font">
+                    Discover the best live comic, perfect 
+                    <br>
+                    <br>
+                    to your comdey type.
+                </p>
+            </section>
+            <section class="header-text-description-container-desk for-desktop">
+                <p class="header-text-description-desk font"> Discover the best live comic, perfect to your comdey type.
+                </p>
+            </section>
+            <section class="T-D">
+                <h2 class="date" >2023.8.15(THU) ~ 16(FRI)</h2>
+                <P class="address">@Rajeeb Gandhi Stadium,Yash Waghmare & More</P>
+            </section>
+        </div>
+    </section>
+    <section id="main" > 
+        <div class="main for-phone">
+            <section>
+                <h3 class="main-head font">Top Specials</h3>
+                <div class="red-line">_____</div>
+            </section>
+            <section class="top-specials-container ">
+                <div class="top-specials-item">
+                    <img class="comic" src="resources/icons/comic-2.png" alt="A Headphone" role="img">
+                    <h3 class="number font">Kaksha Gyarvi</h3>
+                    <section class="special-info-div">
+                        <p>
```

Comment:
Great job on incorporating nearly all the suggested changes from the previous reviewer! 👏 I just wanted to bring up a couple of remaining points that haven't been addressed yet:

- [ ] Please take a moment to verify that the cards within this section are not properly aligned in the desktop version.

Thank you so much for your efforts! 😊🙌

   ### Your design:
   ![image](https://github.com/Shahid-Bagwan/ComedyBurst/assets/107952724/f2a759c8-4f4c-43f1-8afb-55778c97429e)

   ### Figma design:
   ![image](https://github.com/Shahid-Bagwan/ComedyBurst/assets/107952724/c229e426-c70e-4a33-b53c-29d48a95f8a0)




### Your design:### Your design: